src/S3ClientGUI/s3treemodel.cpp

```diff
@@ -20,6 +20,23 @@
 
 #include "s3treemodel.h"
 
+namespace
+{
+	static void advanceSpinner(QVariantList& loadData)
```

Comment:
A simple QString could do this, why use QVariantList ?

Reply:
Because I was used QString ,it was had wrong